module/gui/Options.py

```diff
@@ -0,0 +1,1500 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+from PyQt4.QtCore import *
+from PyQt4.QtGui import *
+
+from os.path import join
+
+from module.gui.Tools import whatsThisFormat, WtDialogButtonBox, LineView
+
+class NotificationOptions(QDialog):
+    """
+        notification options dialog
+    """
+    def __init__(self, parent):
+        QDialog.__init__(self, parent)
+        self.log = logging.getLogger("guilog")
+
+        self.settings = {}
+
+        self.setAttribute(Qt.WA_DeleteOnClose, False)
+        self.setWindowFlags(self.windowFlags() & ~Qt.WindowContextHelpButtonHint)
+        self.setWindowTitle(_("Options"))
+        self.setWindowIcon(QIcon(join(pypath, "icons", "logo.png")))
+
+        self.cbPackageAdded       = QCheckBox(_("Package Added"))
+        self.cbPackageFinished    = QCheckBox(_("Package Download Finished"))
+        self.cbFinished           = QCheckBox(_("Download Finished"))
+        self.cbOffline            = QCheckBox(_("Download Offline"))
+        self.cbSkipped            = QCheckBox(_("Download Skipped"))
+        self.cbTempOffline        = QCheckBox(_("Download Temporarily Offline"))
+        self.cbFailed             = QCheckBox(_("Download Failed"))
+        self.cbAborted            = QCheckBox(_("Download Aborted"))
+        self.cbCaptcha            = QCheckBox(_("New Captcha Request"))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['QCheckBox' may be undefined, or defined from star imports: PyQt4.QtCore, PyQt4.QtGui](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=567903)